TTHAnalysis/python/plotter/susy-interface/scan/masspoints/list_T2ttDiLep.txt

```diff
@@ -0,0 +1,184 @@
+600 : 520 : T2ttDiLep_600_520
```

Comment:
Both, the txt files and q2acc files are analysis dependent. The txt files contain the actual masspoints / samples that one is using and is not a general list of mass points per interpretation. The q2acc files contains histograms of the q2 acceptance shape uncertainty in specific signal regions (e.g. 2lss EWKino). As I understood the TTHAnalysis/data directory, we should keep more general content (e.g. lepton SF) there, potentially useful to a number of different analyses. So I suggest keeping these files where they are.